### PR TITLE
something wrong

### PR DESCRIPTION
**Pull request by VadymBryvka**
[DEMO](https://vadymbryvka.github.io/react_tabs-js/)

### CODE REVIEW

File: src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -1 +1,26 @@
...
+export const Tabs = ({ setTab, tabs, currentTab }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => (
+            <li
+              className={`${currentTab === tab.id ? 'is-active' : ''}`}
+              data-cy="Tab"
+            >
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:10]: Don't forget about key attribute

File: src/components/Tabs/Tabs.jsx [hunk 2]
```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+export const Tabs = ({ setTab, tabs, currentTab }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab, index) => (
+            <li
+              className={`${currentTab === tab.id ? 'is-active' : ''}`}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:8]: Use the classnames library for add classes with condition

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -1 +1,26 @@
...
+              data-cy="Tab"
+            >
+              <a
+                href={tab.id}
+                data-cy="TabLink"
+                onClick={() => {
+                  setTab(index);
+                }}
+              >
+                {tab.title}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:18]: Use destruction for `tab`

File: src/components/Tabs/Tabs.jsx [hunk 4]
```diff
@@ -1 +1,26 @@
...
+            <li
+              className={`${currentTab === tab.id ? 'is-active' : ''}`}
+              data-cy="Tab"
+            >
+              <a
+                href={tab.id}
+                data-cy="TabLink"
+                onClick={() => {
+                  setTab(index);
+                }}
```
Comment (volodymyr-soltys97) [inline src/components/Tabs/Tabs.jsx:16]:
```suggestion
                onClick={() => setTab(index)}
```

File: src/App.jsx [hunk 5]
```diff
@@ -1,19 +1,32 @@
...
-    <div data-cy="TabsComponent">
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${tab.title}`}</h1>
+      <Tabs
+        tabs={tabs}
+        currentTab={tab.id}
+        setTab={fff => {
+          setTab(tabs[fff]);
+        }}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:24]:
```suggestion
        setTab={setTab}
```

File: src/App.jsx [hunk 6]
```diff
@@ -39,6 +52,7 @@ export const App = () => (
       <div className="block" data-cy="TabContent">
         Some text 1
       </div>
+    </div> */}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:55]: Remove comments

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Hi, there is no demo link, also you need to add demo link to the PR description
image

File: src/components/Tabs/Tabs.jsx [hunk 7]
```diff
@@ -1 +1,39 @@
...
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find(ooo => ooo.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
```
Comment (loralevitska) [inline src/components/Tabs/Tabs.jsx:10]:
```suggestion
          {tabs.map({ id, title, ..... }=> (
```

image

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**